README.md

```diff
@@ -1,57 +1,72 @@
-# Getting Started with Wist
+# Wist
 
-Wist is a tool for identifying and reporting on patterns found in BrightScript code, with the goal of making code more consistent and avoiding bugs.
+<p align="center">
```

Comment:
If we're doing inline styling it's highly preferable to do `<p style="text-align: center;">` here.